src/FirmwarePlugin/APM/APMFirmwarePlugin.h

```diff
@@ -137,4 +137,7 @@ class APMFirmwarePluginInstanceData : public QObject
 
     QTime lastBatteryStatusTime;
     QTime lastHomePositionTime;
+
+    bool  MAV_CMD_DO_REPOSITION_supported;
```

Comment:
@peterbarker Are these getting zero initialized someplace else already?

Reply:
My bad.

We zero all allocations as a matter of course in ArduPilot, so it's going to be a blind spot for me :-)

I've zeroed these here in the header now.